app/controllers/application_controller.rb

```diff
@@ -14,20 +14,78 @@ class ApplicationController < Sinatra::Base
   def serialization(weather_response, token)
     if weather_success?(weather_response)
       forecast = Forecast.new(weather_response)
-      spotify_response = SpotifyService.new.playlist(token, forecast.main_description)
-      playlist = spotify_response[:playlists][:items].shuffle.first
-      if playlist_success?(spotify_response)
-        WeatherMusicSerializer.new(forecast, Playlist.new(playlist)).data_hash.to_json
-      else
-        WeatherMusicSerializer.new.no_playlist_response.to_json
+
+      response = SpotifyService.new.weather_tracks_first_50(token)
+      song_ids = response[:items].map {|item| item[:track][:id]}.shuffle
+      five_ids = song_ids.take(5)
+      seed_tracks = five_ids.join(",")
+
+      result = SpotifyService.new.create_playlist(target_valence(forecast), target_speech(forecast), target_mode(forecast), target_energy(forecast), target_tempo(forecast), seed_tracks, token)
```

Comment:
This is awesome! Can't wait to refactor this. Maybe we can break up this large method into smaller helper methods.